Vagrantfile

```diff
@@ -47,7 +237,9 @@ Vagrant.configure("2") do |cluster|
         "--memory", "2048",
```

Comment:
`--name` fails for me when I cloned the fork to other folder and tried to have a separate set of VMs.
I would suggest adding support for some (configurable) prefixes.

Reply:
I'll let @bertvv and @dagwieers and others weigh in on this idea. Perhaps an ENV variable with a default value? Again, we want to err on the side of making this as simple and "foolproof" as possible to get up and running. Our goal is to teach Ansible automation skill, not Vagrant ones. ;)

Reply:
Right, env var or config with defaults is what I'd expect. Consider supporting dotenv in case of env vars.